From 3c6581ae22c0af3abda7803ac564c38ab87a0714 Mon Sep 17 00:00:00 2001
From: Jia Wang <wangjia@ultrarisc.com>
Date: Tue, 7 Apr 2026 10:40:53 +0800
Subject: [PATCH 197/275] FROMLIST: MAINTAINERS: Add entry for the UltraRISC
 DP1000 PCIe controller driver and its DT binding

Add a MAINTAINERS entry for the UltraRISC DP1000 PCIe host driver and its
DT binding.

Signed-off-by: Jia Wang <wangjia@ultrarisc.com>
Link: https://lore.kernel.org/r/20260407-ultrarisc-pcie-v2-2-2aa2a19a7fb3@ultrarisc.com
Signed-off-by: Han Gao <gaohan@iscas.ac.cn>
---
 MAINTAINERS | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/MAINTAINERS b/MAINTAINERS
index 3149fb9f2ab9..62b71c2c9b1e 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -20378,6 +20378,14 @@ S:	Maintained
 F:	Documentation/devicetree/bindings/pci/starfive,jh7110-pcie.yaml
 F:	drivers/pci/controller/plda/pcie-starfive.c
 
+PCIE DRIVER FOR ULTRARISC DP1000
+M:	Xincheng Zhang <zhangxincheng@ultrarisc.com>
+M:	Jia Wang <wangjia@ultrarisc.com>
+L:	linux-pci@vger.kernel.org
+S:	Maintained
+F:	Documentation/devicetree/bindings/pci/ultrarisc,dp1000-pcie.yaml
+F:	drivers/pci/controller/dwc/pcie-ultrarisc.c
+
 PCIE ENDPOINT DRIVER FOR QUALCOMM
 M:	Manivannan Sadhasivam <mani@kernel.org>
 L:	linux-pci@vger.kernel.org
-- 
2.47.3

